packages/client/src/Components/PaperEdits/PaperCutsPlayer/Player.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import VideoContext from 'videocontext';
+
+const getCanvas = (ref, width, height) => {
+  return (
+    <canvas
+      ref={ ref }
+      className='video-canvas'
+      width={ width }
+      height={ height }
+    >
+    </canvas>
+  );
+};
+
+class PaperCutsPlayer extends React.PureComponent {
+  constructor(props) {
+    const {
+      width = 640,
```

Comment:
no, this won't work like that - it's not a CSS styling, it's the sizing for the canvas. In reality, the width and height of the component should be explicitly sent where the component is used. These are just default values so that it doesn't break horribly if you forget.